src/Tracker.js

```diff
@@ -936,6 +998,7 @@ class Tracker extends React.Component {
             itemClicked: true,
             trackerItems: this.setItemState(item, this.state.trackerItems[item] < this.state.max[item] ? this.state.trackerItems[item] + 1 : 0),
         });
+        console.log(item + this.state.trackerItems[item]);
```

Comment:
Avoid leaving console logs in production ready PRs. IO is super slow and can slow down the tracker dramatically when over used. This is mostly just a good practice (and hopefully we can automate it)